config/crd/patches/webhook_in_verticareplicators.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: verticareplicators.vertica.com
+spec:
+  conversion:
+    strategy: Webhook
```

Comment:
We don't need a conversion webhook as there is only v1beta1.

Reply:
will change the strategy to `None`